clang/tools/clang-linker-wrapper/OffloadWrapper.cpp

```diff
@@ -458,44 +459,39 @@ void createRegisterFatbinFunction(Module &M, GlobalVariable *FatbinDesc,
   DtorFunc->setSection(".text.startup");
 
   // Get the __cudaRegisterFatBinary function declaration.
-  auto *RegFatTy = FunctionType::get(PointerType::getUnqual(C)->getPointerTo(),
-                                     PointerType::getUnqual(C),
+  auto *RegFatTy = FunctionType::get(PtrTy, PtrTy,
                                      /*isVarArg*/ false);
```

Comment:
Can you fix these while you're at it, they should use the LLVM-style block comments.
```suggestion
                                     /*isVarArg=*/false);
```